Doc/whatsnew/3.7.rst

```diff
@@ -556,6 +556,18 @@ Other CPython Implementation Changes
 Deprecated
 ==========
 
+* Yield expressions (both ``yield`` and ``yield from`` clauses) are now deprecated
+  in comprehensions and generator expressions (aside from the outermost iterable
+  defined in the leftmost :keyword:`for` clause). This ensures that comprehensions
```

Comment:
I'd drop "defined" here.

Reply:
Based on the changes I made to the language reference, I replaced "the outermost iterable defined in" with "the iterable expression in" here.